src/NServiceBus.Core/Transports/Msmq/MsmqQueueCreator.cs

```diff
@@ -92,6 +96,7 @@ static void CreateQueue(string queuePath, string account, bool transactional)
             }
 
             Logger.DebugFormat("Created queue, path: [{0}], account: [{1}], transactional: [{2}]", queuePath, account, transactional);
+            Logger.WarnFormat("Queue created with [{0}] and [{1}] permissions. Consider changing those, as required", MsmqConstants.LocalEveryoneGroupName, MsmqConstants.LocalAnonymousLogonName);
```

Comment:
@johnsimons thats why the issues says 

> Read the permissions at startup. if anon and everyone is enabled then write to the log. Info if the debugger is attached; otherwise warn

https://github.com/Particular/FeatureDevelopment/issues/303